.github/workflows/python-lint.yml

```diff
@@ -22,3 +22,4 @@ jobs:
         with:
             ignore: "E203,E211,E221,E225,E231,E241,E251,E261,E266,E501,E701,E704,W503"
             exclude: "examples/*,examples/*/**,*/**/__init__.py"
```

Comment:
Does the GitHub runner not honor the flake8 configuration file (`.flake8`)?

Reply:
I'm unsure, hence I'm not touching python-lint.yml ignore or exclude list.

I'm only updating .flake8 so that I can keep to the check on my side (as pre-commit isn't working for some reason) by directly calling `flake8`.

Reply:
You could try removing it and see if the CI still passes on this PR.

Reply:
Well looks like it didn't break even after removing the ignores from this files etc... so looks like the .flake8 addition is taking up the slack now.